core/backend/src/SchemaUtils.ts

```diff
@@ -3,15 +3,14 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-/** @packageDocumentation
- * @module Schema
- */
-
 import { IModelJsNative } from "@bentley/imodeljs-native";
 import { BentleyStatus } from "@itwin/core-bentley";
 import { IModelError } from "@itwin/core-common";
 import { ECSchemaXmlContext } from "./ECSchemaXmlContext";
 
+/** @packageDocumentation
```

Comment:
You were correct before, it should go above imports. Perhaps with your export added to the barrel file, and this reverted, it will work.

Reply:
Ah, I looked at the one other file in this repo which places it below the imports.. Thanks

Reply:
We'll see. I'm running locally as well...